### ISSUE TITLE
Ads on the first tile.

### ISSUE THREAD

**Issue by Kneit0r** [webosbrew/youtube-webos#53]
Since a little while now, there are ads appearing intermittently on the first tile of the top row, with no way to disable or block them.

**Comment by fergardi** [comment]
Yes, I can confirm this is happening to me as well. Seems to be a random issue. Or maybe those are new ads that are still meant to be blocked.
